component_event/tests/test_event.py

```diff
@@ -21,31 +20,9 @@
 class TestEventWorkContext(unittest.TestCase):
     """Test Events Components"""
 
-    if sys.version_info < (3, 8):
-        # Copy/paste from
-        # https://github.com/odoo/odoo/blob/0218d870d319af4f263d5e9aa324990f7cc90139/
-        # odoo/tests/common.py#L248-L268
-        # Partial backport of bpo-24412, merged in CPython 3.8
-        _class_cleanups = []
-
-        @classmethod
-        def addClassCleanup(cls, function, *args, **kwargs):
-            """Same as addCleanup, except the cleanup items are called even if
-            setUpClass fails (unlike tearDownClass). Backport of bpo-24412."""
-            cls._class_cleanups.append((function, args, kwargs))
-
-        @classmethod
-        def doClassCleanups(cls):
-            """Execute all class cleanup functions.
-            Normally called for you after tearDownClass.
-            Backport of bpo-24412."""
-            cls.tearDown_exceptions = []
-            while cls._class_cleanups:
-                function, args, kwargs = cls._class_cleanups.pop()
-                try:
-                    function(*args, **kwargs)
-                except Exception:
-                    cls.tearDown_exceptions.append(sys.exc_info())
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.test_sequence = 0
```

Comment:
I didn't meet such code before. Could you pls explain what `self.test_sequence = 0` is for?

Reply:
@i-vyshnevska ?

Reply:
the field is defined in odoo metatest class
 https://github.com/odoo/odoo/blob/5cfce06bf92cc746b4dc3b2b554ad57daac1af1d/odoo/tests/common.py#L296
 but us we inherit here form uittetst we also need this field so odoo will be able to run test

Reply:
@i-vyshnevska thanks for the explanation